palace/linalg/arpack.cpp

```diff
@@ -682,25 +713,20 @@ int ArpackPEPSolver::Solve()
   if (!eig)
   {
     eig = std::make_unique<std::complex<double>[]>(nev + 1);
-    perm = std::make_unique<int[]>(nev + 1);
-    res = std::make_unique<double[]>(nev + 1);
+    perm = std::make_unique<int[]>(nev);
```

Comment:
Why the change from `nev+1` to `nev`, was it previously oversized?

Reply:
Yep, only the eigenvectors need to be `eig + 1` for ARPACK. Everything else is sized for `nev` and used by on the Palace side.